### PR TITLE
Proposed launch commit for new themes and bootstrap responsive conversion by zbig01.

### PR DESCRIPTION
**Pull request by bradymiller**
This is the proposed initial commit to bring in zbig01's new themes, remove old themes, and brings in a converted SOAP form (after this commit, plan to then bring in all the other upgraded scripts, of which there are only about 70 or so so far :) ).
@zbig01 
Code taken from following PR:
https://github.com/openemr/openemr/pull/1076

### CODE REVIEW

File: interface/themes/core.css [hunk 1]
```diff
@@ -89,14 +89,22 @@ button[href*="deleter"]:before {
 .btn-undo::before {
     content: '\f0e2\00a0\00a0';
 }
+.btn-update::before {
```
Comment (bradymiller) [inline interface/themes/core.css]: TODO:
There is already an update button in this script. Which one to use? @zbig01

**Comment by zbig01** [comment]
@bradymiller 
 The following font-awesome icons are used in the 2 update buttons

```
.btn-update::before {
    content: '\f021\00a0\00a0'; -  fa-refresh
}
.btn-update::before {
    content: '\f01e\00a0\00a0'; - fa-repeat
}
```
The latter one is a clockwise circular arrow. The other consists of 2 semicircular arrows.

 Interestingly this icon also represents the refresh button in core.css

```
.btn-refresh::before { - fa-refresh
    content: '\f021\00a0\00a0';
}
```
I think using ''\f01e\00a0\00a0'; - fa-repeat  would avoid different actions being represented by the same icon.

**Comment by bradymiller** [comment]
hi @zbig01 ,
Thanks for the rec and just removed the duplicate update and set it to f01e .
-brady

**Comment by bradymiller** [comment]
@zbig01 ,
This code is testing well and plan to bring it into the codebase soon. Just let me know if you have any issues with it.
thanks, -brady

File: interface/themes/core.css (hunk 1 above)
Comment (bradymiller) [reply interface/themes/core.css]: FIXED

**Comment by bradymiller** [comment]
testing well, so bringing into codebase.
thank you @zbig01 !! And looking very forward to getting all your bootstrapped scripts into the codebase!
-brady

**Pull request merged**